Formula/weechat.rb

```diff
@@ -3,7 +3,8 @@ class Weechat < Formula
   homepage "https://www.weechat.org"
   url "https://weechat.org/files/src/weechat-2.9.tar.xz"
   sha256 "eab406c385c3a10d0107ddc3aac6596ae8c59af99e9158c6d769e90ec9adfa0e"
-  license "GPL-3.0"
+  license "GPL-3.0-only"
```

Comment:
This one also looks like
```suggestion
  license "GPL-3.0-or-later"
```

From the [README](https://github.com/weechat/weechat#copyright):

> Copyright © 2003-2020 Sébastien Helleu <flashcode@flashtux.org>
>
>This file is part of WeeChat, the extensible chat client.
>
>WeeChat is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation; either version 3 of the License, or (at your option) any later version.
>
>WeeChat is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
>
> You should have received a copy of the GNU General Public License along with WeeChat. If not, see https://www.gnu.org/licenses/.